docker-compose.yml

```diff
@@ -46,6 +46,7 @@ services:
       - COMMS_MODE=archipelago
       - STORAGE_LOCATION=/app/storage
       - REALM_NAMES=${LIGHTHOUSE_NAMES}
+      - MAX_USERS=300
```

Comment:
I think we should at least put the option to override it in the env, so people can override it without having to change the docker-compose

Reply:
I think this new approach should work, anyways I'll test it in a peer-testing node.